inference/server/oasst_inference_server/user_chat_repository.py

```diff
@@ -108,17 +108,12 @@ def initiate_assistant_message(self, parent_id: str, work_parameters: inference.
         self.session.refresh(message)
         return message
 
-    def add_vote(self, message_id: str, score: int) -> models.DbVote:
-        logger.info(f"Adding vote to {message_id=}: {score=}")
+    def update_score(self, message_id: str, score: int):
```

Comment:
If we are returning message can we add the type hint here as `models.DbMessage`

Reply:
Done!